src/session.ts

```diff
@@ -228,3 +228,39 @@ export async function sendFeedback(body: Feedback) {
 
   return responseJson;
 }
+
+export interface UpdateChainRequest {
```

Comment:
Should this be titled `UpdateChainResult` to better reflect what it actually does?